tango/device_server.py

```diff
@@ -198,23 +198,24 @@ def __init__(self):
 
 
 def __Attribute__get_properties(self, attr_cfg=None):
-    """get_properties(self, attr_cfg = None) -> AttributeConfig
-
-                Get attribute properties.
-
-            Parameters :
-                - conf : the config object to be filled with
-                         the attribute configuration. Default is None meaning the
-                         method will create internally a new AttributeConfig_5
-                         and return it.
-                         Can be AttributeConfig, AttributeConfig_2,
-                         AttributeConfig_3, AttributeConfig_5 or
-                         MultiAttrProp
+    """
+    get_properties(self, attr_cfg = None) -> AttributeConfig
 
-            Return     : (AttributeConfig) the config object filled with
-                         attribute configuration information
+        Get attribute properties.
 
-            New in PyTango 7.1.4
+        Parameters:
```

Comment:
It looks like the first half of the file is using the google style formatting, while most of the docstrings using `document_method` are using reST.  Is there a reason for the inconsistency?

Reply:
Done